### ISSUE TITLE
Is Project Calico open to taking over confd

### ISSUE THREAD

**Issue by retornam** [projectcalico/confd#342]
@caseydavenport and the Project Calico team, thanks for actively maintaining this fork of confd.
Based on the twitter discussion, Kelsey is open to officially handing over confd to Project Calico, if you 
are interested.

 
<img width="604" alt="Screen Shot 2020-07-13 at 2 27 57 PM" src="https://user-images.githubusercontent.com/179895/87355327-0e1d2e80-c515-11ea-9f96-d8b2d2b36522.png">

https://twitter.com/kelseyhightower/status/1282786865238536192?s=20

**Comment by kelseyhightower** [comment]
I'm onboard with this.

**Comment by caseydavenport** [comment]
Hey guys!

Thanks for reaching out. I'm flattered by the offer and I really _want_ to be able to help the community on this one. But I'm not convinced it really makes sense for us to take over the project at this point in time. Calico's fork of confd has actually diverged quite a bit from the upstream over time. As an example - we used to rely on the etcd backend, but we eventually wrote a backend based off of libcalico-go in order to satisfy some of our other needs, and now this fork doesn't actually even include any other backends besides the Calico backend.

So, I think given this, we're probably not the right folks to really satisfy the community's need here.
